eng/.docsettings.yml

```diff
@@ -126,18 +126,15 @@ known_presence_issues:
 known_content_issues:
   - ['README.md','Root readme']
   - ['doc/README.md','Doc readme']
-  - ['sdk/anomalydetector/Azure.AI.AnomalyDetector/README.md', '#5499']
   - ['sdk/core/Azure.Core/README.md', '#5499']
   - ['sdk/core/Azure.Core.Experimental/README.md', '#5499']
   - ['sdk/core/Azure.Core.Amqp/README.md', '#5499']
   - ['sdk/core/System.Memory.Data/README.md', '#5499']
   - ['sdk/keyvault/Microsoft.Azure.KeyVault/README.md','#5499']
   - ['sdk/servicebus/Microsoft.Azure.ServiceBus/README.md','#5499']
   - ['sdk/eventhub/Microsoft.Azure.EventHubs/README.md','#5499']
-  - ['sdk/eventhub/Azure.Messaging.EventHubs.Shared/README.md','#5499']
```

Comment:
Unless they changed part of the analysis recently, I believe we'll still need this.   `Shared` is not a shipping package, so the README isn't structured like a typical client library file.

Reply:
Yep, I was planning to replace the issue tracking number with an explanation of that, and keep the exception in there.  Let me know if you have any concerns about this!

Reply:
Works for me.   Apologies for the drive-by.   😄

Reply:
Oh no worries, thanks for your input!  😊🌷